uui/components/datePickers/Calendar.module.scss

```diff
@@ -5,15 +5,15 @@
     --uui-calendar-selected_range: var(--uui-primary-50);
     --uui-calendar-weekday-caption: var(--uui-text-tertiary);
     --uui-calendar-holiday-caption: var(--uui-text-tertiary);
-    --uui-calendar-day-caption: var(--uui-control-text);
-    --uui-calendar-day-caption-disabled: var(--uui-control-text-disabled);
+    --uui-calendar-day-text: var(--uui-control-text);
+    --uui-calendar-day-text-disabled: var(--uui-control-text-disabled);
+    --uui-calendar-selected_day-caption: var(--uui-neutral-0);
```

Comment:
i guess --uui-calendar-selected_day-text will be more align with other variables